iOSTest/View Controllers/LoginViewController.swift

```diff
@@ -28,19 +28,75 @@ class LoginViewController: UIViewController {
     // MARK: - Properties
     @IBOutlet weak var usernameTextfield: UITextField!
     @IBOutlet weak var passwordTextfield: UITextField!
+    
+    @IBOutlet weak var usernameView: UIView!
+    @IBOutlet weak var passwordView: UIView!
+    
     private var client = LoginClient()
+    var username:String?
+    var password:String?
     
     // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
         
         title = "Login"
+        roundViewCorners()
     }
     
     // MARK: - Actions
     
     @IBAction func didPressLoginButton(_ sender: Any) {
         
+        let startTime = CACurrentMediaTime()
+        
+        username = usernameTextfield.text
+        password = passwordTextfield.text
+        
+        validateUsernameAndPassword()
+        
+        let endTime = CACurrentMediaTime()
+        let runtime = endTime - startTime
+        print("Runtime: \(runtime)")
+        
     }
     
+    // MARK: - Additional Functions
+    
+    func roundViewCorners(){
+        usernameView.layer.cornerRadius = 5.0
+        passwordView.layer.cornerRadius = 5.0
+    }
+    
+    func validateUsernameAndPassword() {
+
+        let validUsername = "user@iosstudies.co"
+        let validPassword = "pinksalmon"
```

Comment:
These properties can be assigned as global properties in this view controller. Move them underneath line 37.